config.go

```diff
@@ -475,6 +478,7 @@ func DefaultConfig() Config {
 		},
 		MaxOutgoingCltvExpiry:   htlcswitch.DefaultMaxOutgoingCltvExpiry,
 		MaxChannelFeeAllocation: htlcswitch.DefaultMaxLinkFeeAllocation,
+		MaxCommitFeeRateAnchors: lnwallet.DefaultAnchorsCommitMaxFeeRateSatPerVByte,
```

Comment:
So for the itests, I think we can just not set the value (somehow?) to have it ignore if we want to reduce the surface area of tests we'll need to change.